### PR TITLE
made the badge work with list-group

### PR DESCRIPTION
**Pull request by brianbodollo**

### CODE REVIEW

**Comment by srph** [comment]
<sup>Don't forget to reference the issue next time.</sup> :smile:

Addresses https://github.com/dev-onenetworkecommerce/aia/pull/63

How about the docs? :smile:

**Comment by brianbodollo** [comment]
badge

**Comment by srph** [comment]
Ooooh, looks good, man. I'm not sure if it's just me &mdash; the badge isn't vertically aligned (properly), is it?

**Comment by brianbodollo** [comment]
its vertically align on me

**Comment by srph** [comment]
The badge isn't.

**Comment by brianbodollo** [comment]
should i use margin in this case?

**Comment by srph** [comment]
Nah, leave it as it is for now. I'll put an issue

**Comment by brianbodollo** [comment]
all are set then.

File: styles/components/list-group.scss [hunk 1]
```diff
@@ -37,6 +37,10 @@
     background-color: $brand-primary;
   }
 
+  > .menuitem > .badge {
```
Comment (srph) [inline styles/components/list-group.scss]: Can you place this rules (`.badge`) after the rules for the icon?

**Comment by srph** [comment]
Badges should be colored `$brand-primary` by default (no need to specify). When the link is hovered or selected, the badge it should be `$brand-white` with a `$brand-black` text

**Comment by brianbodollo** [comment]
okay, just a sec

**Comment by brianbodollo** [comment]
badge

good to go?

**Comment by srph** [comment]
Looks good, man

**Pull request merged**